plexapi/config.py

```diff
@@ -60,4 +60,5 @@ def reset_base_headers():
         'X-Plex-Device': plexapi.X_PLEX_DEVICE,
         'X-Plex-Device-Name': plexapi.X_PLEX_DEVICE_NAME,
         'X-Plex-Client-Identifier': plexapi.X_PLEX_IDENTIFIER,
+        'X-Plex-Sync-Version': '2',
```

Comment:
I've added this because it doesn't require anything from the client, it just reports to the server the we can work with Sync. Basically without this header you can't add new SyncItems.